inst/WORDLIST

```diff
@@ -37,25 +112,17 @@ createPlotTicks
 createWatermarkGrob
 csv
 dataMapping
-DataMapping
 dataMappingLabel
-DDI
-DDIRatioDataMapping
-DDIRatioPlotConfiguration
-DefaultDataMappingValues
 dodgerblue
 enum
-Enum
 enums
 errorbar
 errorbars
 et
-ExportConfiguration
-ExportFormats
 exportPlot
 exportPlotConfigurationCode
-ExportUnits
-FontFaces
+fonctions

```

Comment:
that's a typo which should be fixed rather than entered into the word list

Reply:
French bias here. Fixed in next commit.